apps/sched/sched.js

```diff
@@ -1,10 +1,81 @@
+/* eslint-disable no-unused-vars */
+
 // Chances are boot0.js got run already and scheduled *another*
 // 'load(sched.js)' - so let's remove it first!
 if (Bangle.SCHED) {
   clearInterval(Bangle.SCHED);
   delete Bangle.SCHED;
 }
 
+function formatMS(ms) {
+  if (ms < 60000) {
+    // less than a minute → show seconds
+    return Math.round(ms / 1000) + "s";
+  } else {
+    // one minute or more → show minutes
+    return Math.round(ms / 60000) + "m";
+  }
+}
+
+function showSnoozeMenu(alarm){
+  const alarmIndex = alarms.indexOf(alarm);
+  const settings = require("sched").getSettings();
+  let buzzCount = 0;
+  let buttons=
+  Bangle.buzz(40);
```

Comment:
I imagine this is one of the unused:

```suggestion
  Bangle.buzz(40);
```